app/controllers/events_controller.rb

```diff
@@ -0,0 +1,22 @@
+class EventsController < ApplicationController
+  def new
+    @event = Event.new
+  end
```

Comment:
## Changes Required
### Hi guys :wave:  
I have been able to sign up successfully.
When I try to create a new event, I get this error.
![sknd](https://user-images.githubusercontent.com/44072711/72210782-a7985480-348e-11ea-9b5f-0086fd36af24.png)